keepassxc-browser/options/options.html

```diff
@@ -518,6 +518,7 @@ <h2 class="pb-3 mt-0" data-i18n="optionsSitePreferencesTab"></h2>
                             <select class="form-control form-control-sm" name="ignore" data-i18n="[title]optionsSitePreferencesSelect">
                               <option value="ignoreNothing" data-i18n="optionsSelectionNothing"></option>
                               <option value="ignoreNormal" data-i18n="optionsSelectionNormal"></option>
+                              <option value="ignoreAutoSubmit" data-i18n="optionsSelectionAutoSubmit"></option>
```

Comment:
I think we have outgrown a signal choice list 🤔
We can keep this change for now but we need to rethink this workflow.

Reply:
I was thinking exactly the same. This is the maximum options we can have as a combo box. Anything extra would need a redesign.